src/components/AddStory.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./AddStory.css";
+import { Button, TextField, Divider, Box } from "@material-ui/core";
+import {
+  createMuiTheme,
+  ThemeProvider,
+} from '@material-ui/core/styles';
+
+const AddStory = () => {
+
+  const theme = createMuiTheme({
+    palette: {
+      primary: {
+        main: "#00695c",
+        contrastText: '#fff'
+      }
+    }
+  })
+  
+
+  return (
+      <main>
+        <Box display="flex" justifyContent="space-between" mt="1rem">
+          <h1>Add Story</h1>
+          <Button variant="outlined">Cancel</Button>
+        </Box>
+        <form>
+        <TextField className="url-input"
```

Comment:
Could you also make the text in this text field a placeholder so you don't have to delete the text before adding your own URL?